### PR TITLE
Add db service

### PR DESCRIPTION
**Pull request by ngambino0192**
Convert to postgres service, consolidate .env files, update README

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -1 +1,13 @@
-DOCKER_USER=*****
\ No newline at end of file
+FINNHUB_TOKEN=api_token
+MYSQL_DATABASE=string
+MYSQL_USER=string
+MYSQL_PASSWORD=string
+MYSQL_ROOT_PASSWORD=string
```
Comment (melsoriano) [inline .env.example:5]: Are these still needed since we're using PostgreSQL?

File: .env.example [hunk 2]
```diff
@@ -1 +1,13 @@
...
+DOCKER_USER=string
+MYSQL_PORT=integer
```
Comment (melsoriano) [inline .env.example:7]: Same here. Is this still needed?

**Comment by melsoriano** [comment]
Is the `.env` inside of the `backend` directory still being used?

**Comment by ngambino0192** [comment]
> Is the `.env` inside of the `backend` directory still being used?


Nope

**Pull request merged**